kadi/update_cmds.py

```diff
@@ -105,18 +104,15 @@ def fix_nonload_cmds(nl_cmds):
         if cmd['msid'] is not None:
             new_cmd['params']['msid'] = str(cmd['msid'])
 
-        # De-unicode and de-numpy (otherwise unpickling on PY3 has problems).
+        # De-numpy (otherwise unpickling on PY3 has problems).
         if 'params' in cmd:
             params = new_cmd['params']
             for key, val in cmd['params'].items():
                 key = str(key)
-                if isinstance(val, six.string_types):
```

Comment:
Previous logic was that `val` might be a Py2 `str` or `unicode`, and it was necessary to coerce to `str`.  See the dump above of an example of the data structure.  In Py3 none of this needed and instead we just try using the numpy `item()` method to coerce to pure-Python.